doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -4,6 +4,8 @@
 
 SSL_SESSION_get_time, SSL_SESSION_set_time, SSL_SESSION_get_timeout, SSL_SESSION_set_timeout - retrieve and manipulate session time and timeout settings
 
+SSL_SESSION_get0_hostname - retrieve the hostname for the session
```

Comment:
Side comment: the docs should be built up during build, not during install.  If the rest of you agree, I'll make it so.